snapshot_manager/snapshot_manager/performance_diagrams.py

```diff
@@ -261,7 +261,7 @@ def build_performance_diagrams(datafile: pathlib.Path | str) -> None:
     pio.templates.default = "plotly"
 
     if isinstance(datafile, str):
-        datafile = filepath.Path(datafile)
+        datafile = pathlib.Path(datafile)
```

Comment:
This was a real error before because `filepath` does not exist as a module but `pathlib`.